pkgs/top-level/aliases.nix

```diff
@@ -528,6 +528,7 @@ mapAliases ({
   grub = throw "grub1 was removed after not being maintained upstream for a decade. Please switch to another bootloader"; # Added 2023-04-11
   gtkcord4 = dissent; # Added 2024-03-10
   gtkpod = throw "'gtkpod' was removed due to one of its dependencies, 'anjuta' being unmaintained"; # Added 2024-01-16
+  guardian-agent = throw "'guardian-agent' has been removed, as it hasn't been maintained upstream in years and thus has multiple vulnerabilities"; # Added 2024-06-09
```

Comment:
```suggestion
  guardian-agent = throw "'guardian-agent' has been removed, as it hasn't been maintained upstream in years"; # Added 2024-06-09
```

Technically, we have no evidence of vulnerabilities - like, say, a CVE entry. Let's be less opinionated.

Is there any other alternative software?

Reply:
Oh, there are many CVEs: https://pkg.go.dev/search?q=crypto&m=vuln

Pretty much everything on this page applies to the vendored package.